x/auth/client/utils/tx.go

```diff
@@ -154,8 +154,14 @@ func PrintUnsignedStdTx(txBldr authtypes.TxBuilder, cliCtx context.CLIContext, m
 		return err
 	}
 
-	json, err := cliCtx.Codec.MarshalJSON(stdTx)
-	if err != nil {
+	var json []byte
+	if viper.GetBool(flags.FlagIndentResponse) {
+		json, err = cliCtx.Codec.MarshalJSONIndent(stdTx, "", "  ")
+	} else {
+		json, err = cliCtx.Codec.MarshalJSON(stdTx)
+	}
+	if err == nil {
```

Comment:
Should this be `if err != nil`? Also, why are we printing here?

Reply:
Oh, I must be crazy at that time.